resources/themes/cura-light/theme.json

```diff
@@ -191,6 +191,8 @@
 
         "printer_type_label_background": [228, 228, 242, 255],
 
+        "text_light_blue": [50, 130, 255, 255],
```

Comment:
As said before, I'd like this theme entry to be removed.

Reply:
Yeah, we shouldn't have this.

Reply:
Done in 1e3945810